bin/oneoff/deprecate_unused_poste_urls.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require_relative './poste_urls_constants'
+
+# This script is a dry-run unless we change DRY_RUN to false
+DRY_RUN = true
+
+if DRY_RUN
+  puts "DRY RUN..."
+else
+  puts "Starting update..."
+end
+
+DB.transaction do
```

Comment:
I think this will carry out all of the >3M updates in a single transaction, which would impact database performance at the end when the 3M row updates are committed. Since the update is simple and failures on individual rows are rare, I think we could avoid declaring any transaction at all and allow each individual row update to be carried out implicitly in its own transaction.

Reply:
hmmmm ... I did not chose a good example script for you to reference. The DRY_RUN / rollback logic won't work if we eliminate the transaction logic.

Reply:
Reflecting on this further, since we're not writing to this table anymore and we have 30 days of backups of the table, I think it's safe to not have any DRY_RUN functionality in this script.

Reply:
Ah makes sense! will update